src/components/EmailVerification/EmailVerificationForm.tsx

```diff
@@ -1,27 +1,33 @@
 import {TextField} from '@open-formulieren/formio-renderer';
 import {ButtonGroup} from '@utrecht/button-group-react';
 import {Link as UtrechtLink} from '@utrecht/component-library-react';
-import {Formik} from 'formik';
-import PropTypes from 'prop-types';
+import {Formik, FormikHelpers} from 'formik';
```

Comment:
```suggestion
import {Formik, type FormikHelpers} from 'formik';
```
here it's less relevant because we also use a `value` import of `Formik`, but this demonstrates the syntax :)